css/styles.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Regular.ttf);
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Bold.ttf);
+    font-weight: bold;
+}
+
+body {
+    color: #424242;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 14px;
+    letter-spacing: .01em;
+    line-height: 1.3;
+    padding: 24px;
+    max-width: 800px;
+}
+
+@media screen and (max-width: 960px) {
+    body {
+        padding: 16px;
```

Comment:
I think it's the way I took the screenshot that makes it look big. Here's another screenshot that uses the same scale as the desktop screenshot (in the PR):

![image](https://user-images.githubusercontent.com/159687/39279072-ba963bb8-48ab-11e8-900d-69ac149875d1.png)

... and for a point of comparison, here's what the margins look like on medium.com on phone screens (they use 20px margins):

![image](https://user-images.githubusercontent.com/159687/39279106-f4aee9ee-48ab-11e8-8863-2597a4488c74.png)

So unless you feel strongly, I'd opt to go no lower than 16px (and especially since the font size and line heights more significantly affects amount of text, I'd opt to tweak those if we're concerned about text density).